Matlab/DecisionFunctions/comp_model.m

```diff
@@ -471,28 +472,28 @@
             
         end
         
-        function summary = summary(obj)
+        function summary = summary(obj,varargin)
             % summary = summary(obj)
             %
             % -------------------------------------------------------------------------
             % This function returns the average parameter estimates across
             % subjects.  Assumes that parameters start on the second
             % column. Also returns average AIC, average BIC, average final
-            % minimized value, and number of subjects
+            % minimized value, and number of subjects.
             % -------------------------------------------------------------------------
             
             sprintf(['Summary of Model: ' obj.model ...
                 '\n-----------------------------------------' ...
                 '\nAverage Parameters:\t' num2str(nanmean(obj.params(:,2:length(obj.param_min) + 1))) ...
                 '\nAverage AIC:\t\t' num2str(nanmean(obj.params(:,end - 1))) ...
                 '\nAverage BIC:\t\t' num2str(nanmean(obj.params(:,end))) ...
-                '\nAverage ' obj.esttype ':\t\t' num2str(nanmean(obj.params(:,end-3))) ...
+                '\nAverage ' obj.esttype ':\t\t' num2str(nanmean(obj.params(:,end-2))) ...
```

Comment:
@danweflen: Are you sure this is right?  I vaguely remember fixing a random bug here awhile back.